docs/repositories/commits.md

```diff
@@ -60,61 +64,72 @@ This area displays the quality gate status and an overview of the code quality m
 
--   If a file had [more than 50 issues reported by the same tool](../faq/code-analysis/does-codacy-place-limits-on-the-code-analysis.md) and you push a new commit that fixes some of these issues, Codacy will report more issues until the limit of 50 issues. These issues will be possible issues if they're outside the lines of code changed in the new commit.
+-   If a file had [more than 50 issues reported by the same tool](../faq/code-analysis/does-codacy-place-limits-on-the-code-analysis.md) and you push a new commit that fixes some of these issues, Codacy will report more issues until the limit of 50 issues. These issues will be potential issues if they're outside the lines of code changed in the new commit.
 
 !!! note
-    **If you're using GitHub** you may see [annotations](../repositories-configure/integrations/github-integration.md#issue-annotations)  for possible issues reported under **Unchanged files with check annotations** on the **Files changed** tab of your pull requests.
+    **If you're using GitHub** you may see [annotations](../repositories-configure/integrations/github-integration.md#issue-annotations) for potential issues reported under **Unchanged files with check annotations** on the **Files changed** tab of your pull requests.
+
+    This happens when Codacy reports potential issues in files that weren't changed in your pull request. [Read more about this GitHub feature](https://developer.github.com/changes/2019-09-06-more-check-annotations-shown-in-files-changed-tab/).
+
+## Duplication tab {: id="duplication-tabs"}
 
-    This happens when Codacy reports possible issues in files that weren't changed in your pull request. [Read more about this GitHub feature](https://developer.github.com/changes/2019-09-06-more-check-annotations-shown-in-files-changed-tab/).
+The **Duplication** tab displays the lists of clones (duplicated code blocks) that the {{ page.meta.page_name }} adds or fixes. You can click a clone to expand it and inspect the code.
 
-## Duplication tabs
+![Duplication tabs](./images/{{ page.meta.file_name }}-tab-duplication.png)
 
-The **New Duplication** and **Fixed Duplication** tabs display the list of duplicated blocks that the {{ page.meta.page_name }} created or fixed.
+## Complexity tab
 
-![New Duplication and Fixed Duplication tabs](images/{{ page.meta.file_name }}-tab-duplication.png)
+The **Complexity** tab displays the complexity changes introduced by the {{ page.meta.page_name }}. Use the sidebar filters to filter the list by high increase, low increase, or improvement.
+
+![Complexity tab](./images/{{ page.meta.file_name }}-tab-complexity.png)
+
+{%
+    include-markdown "../repositories-coverage/commits.md"
+    start="<!--tab-diff-start-->"
+    end="<!--tab-diff-end-->"
+%}
```

Comment:
Confirm this is fixed, see [thread](https://codacy.slack.com/archives/C04EJUDAL8J/p1710242162268419?thread_ts=1710173646.391709&cid=C04EJUDAL8J)